packages/steiger-plugin-fsd/src/shared-lib-grouping/index.ts

```diff
@@ -1,13 +1,18 @@
 import { getLayers, getSegments } from '@feature-sliced/filesystem'
-import type { Diagnostic, Rule } from '@steiger/types'
+import type { Diagnostic, Rule, RuleOptions } from '@steiger/types'
 import { NAMESPACE } from '../constants.js'
 
-const THRESHOLD = 15
+const DEFAULT_THRESHOLD = 15
+
+export interface SharedLibGroupingOptions extends RuleOptions {
+  threshold?: number
+}
```

Comment:
issue: I understand that you want to have demo of the rule options capability, I just think that this rule shouldn't be configurable like that, not yet, anyway. Maybe later, after we do some research on the results of this rule.

suggestion: let's remove the configurability of this rule. As for the demo, I don't think we must have it no matter what, I'd prefer if we give it some separate thought.

Reply:
No problem, its main purpose was to let me manually test the new functionality and confirm that it works e2e. Then I decided to button up and commit it.